config/rbac/service_account.yaml

```diff
@@ -90,3 +90,41 @@ subjects:
 - kind: ServiceAccount
   name: openstackprovisionserver
   namespace: openstack
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: openstackplaybookgeneratior
```

Comment:
should we name this openstackplaybookgenerator-role (vs tior-role)

Reply:
good catch, thats a typo. Should be fixed now